site/_config.yml

```diff
@@ -8,6 +8,7 @@ destination: local-generated
 twitter_url: https://twitter.com/asfbookkeeper
 
 versions:
+- "4.8.1"
```

Comment:
can you move this down after `# [next_version_placehoder]`?

I think the script requires some fixes.

Reply:
Sure.
The a script choose this position.
Good catch

Reply:
this is not fixed?